libs/use-multi-media-streams/src/index.ts

```diff
@@ -130,40 +155,54 @@ const useMediaDevices = ({ handleError, filterOutUsedDevices = true }: UseMediaD
   };
 
   const initDefaultStream = () => {
-    if (microphoneList.length > 0 && cameraList.length > 0) {
-      addStream({ camera: cameraList[0], microphone: microphoneList[0], type: StreamTypes.MEDIA });
+    if (microphoneList.length && cameraList.length) {
+      addStream({
+        bitrate: bitrateList[0].value,
```

Comment:
Bitrate and Codec are all conceptions in the `usePublisher` hook, they shouldn't stay here.